Jenkinsfile

```diff
@@ -192,6 +192,56 @@ pipeline {
                         }
                     }
                 }
+                stage('Single node valgrind') {
+                    agent {
+                        label 'ci_vm1'
+                    }
+                    steps {
+                        provisionNodes NODELIST: env.NODELIST,
+                           node_count: 1,
+                           snapshot: true
+                        runTest stashes: [ 'CentOS-install', 'CentOS-build-vars' ],
+                                script: """set -x
+                                    . ./.build_vars-Linux.sh
+                                    CART_BASE=\${SL_PREFIX%/install*}
+                                    NODELIST=$nodelist
+                                    NODE=\${NODELIST%%,*}
+                                    trap 'set +e; set -x; ssh -i ci_key jenkins@\$NODE "set -ex; sudo umount \$CART_BASE"' EXIT
+                                    ssh -i ci_key jenkins@\$NODE "set -x
+                                        set -e
+                                        sudo mkdir -p \$CART_BASE
+                                        sudo mount -t nfs \$HOSTNAME:\$PWD \$CART_BASE
+                                        cd \$CART_BASE
+                                        ln -s /usr/bin/fusermount install/Linux/bin/fusermount3
+                                        pip3.4 install --user tabulate
+					export TR_USE_VALGRIND=memcheck
+					export IOF_TESTLOG=test/output-memcheck
+                                        nosetests-3.4 --xunit-testsuite-name=valgrind --xunit-file=nosetests-valgrind.xml --exe --with-xunit"
+                                    exit 0
+                                    """,
+                        junit_files: 'nosetests-valgrind.xml'
+                        publishValgrind (
```

Comment:
I note that this is done in the `post { }` in CaRT.  That seems like better isolation from steps blowing up the stage and the `publishValgrind` not getting done.

Reply:
Oh yeah, this is wrong isn't it.  Will correct.